llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -743,9 +743,16 @@ getLineTableLengthInfoAndVersion(DWARFDataExtractor &LineTableDataReader,
   auto Version = LineTableDataReader.getU16(OffsetPtr, &Err);
   if (Err)
     return std::move(Err);
-  if (Version >= 5)
-    return createStringError(inconvertibleErrorCode(),
-                             "DWARF 5 and above is not currently supported");
+  if (Version >= 5) {
+    // DWARF v5 6.2.4 read address_size and segment_selector_size
+    LineTableDataReader.getU8(OffsetPtr, &Err);
```

Comment:
Since you are discarding the Address size, shouldn't you at least error out if it isn't 64 bit, which I'm assuming you are implicitly assuming?